### PR TITLE
Add OpenPaC Bluemap - Remapped to addons.conf

### PR DESCRIPTION
**Pull request by DreamyLynn**

### CODE REVIEW

File: assets/addon_browser/addons.conf [hunk 1]
```diff
@@ -603,4 +603,15 @@
       "github": "https://github.com/Chicken/Auth/tree/master/BlueMap/PrivateMaps"
     }
   }
+  {
+    name: "OpenPaC BlueMap - Remapped"
+    description: "Show land claims from the Open Parties and Claims mod as markers in BlueMap."
+    author: "DreamyLynn"
+    apiVersion: "2"
+    platforms: ["fabric", "quilt"]
```
Comment (TechnicJelle) [inline assets/addon_browser/addons.conf:611]: `quilt` is not a valid platform for this config at the moment..
I think it's clear that it supports quilt if fabric itself is already mentioned :)

**Review by TechnicJelle — CHANGES_REQUESTED** [review]

File: assets/addon_browser/addons.conf (hunk 1 above)
Comment (Chicken) [reply assets/addon_browser/addons.conf:611]: You can also just add an icon for quilt in `/assets/addon_browser/platforms/`

Comment (DreamyLynn) [reply assets/addon_browser/addons.conf:611]: added an icon for quilt

**Comment by DreamyLynn** [comment]
It's always worth knowing if the maintainer cares about quilt and fixes quilt-specific issues. I added an icon with the same resulution as the other ones in the folder :).

**Pull request merged**